### PR TITLE
New Keyboard: J-01

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Submitting a new keyboard from Jae called the J-01, only 15 made. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jae/j01/config.h [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+#define LCD_RW_PORT      LCD_PORT     //< port for RW line
+#define LCD_RW_PIN       2            //< pin  for RW line
+#define LCD_E_PORT       LCD_PORT     //< port for Enable line
+#define LCD_E_PIN        1            //< pin  for Enable line
+#endif
+*/
+
+/* Bootmagic Lite key configuration */
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 1
```
Comment (drashna) [inline keyboards/jae/j01/config.h]:
```suggestion
#define BOOTMAGIC_LITE_COLUMN 2
```

**Comment by mechmerlin** [comment]
> Where is the ISO layout? :D

Never heard of it.

File: keyboards/jae/j01/readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# J-01
+
+A 65% with the arrow keys and right most column exploded away, with macro keys on the left hand side. Only 15 manufactured. 
+
+* Keyboard Maintainer: [Maartenwut](https://github.com/maartenwut)
```
Comment (fauxpark) [inline keyboards/jae/j01/readme.md]:
```suggestion
* Keyboard Maintainer: [Maartenwut](https://github.com/maartenwut)
```

File: keyboards/jae/j01/readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# J-01
+
+A 65% with the arrow keys and right most column exploded away, with macro keys on the left hand side. Only 15 manufactured. 
+
+* Keyboard Maintainer: [Maartenwut](https://github.com/maartenwut)  
+* Hardware Supported: J-01 PCB
```
Comment (fauxpark) [inline keyboards/jae/j01/readme.md]:
```suggestion
* Hardware Supported: J-01 PCB
```

File: keyboards/jae/j01/readme.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# J-01
+
+A 65% with the arrow keys and right most column exploded away, with macro keys on the left hand side. Only 15 manufactured. 
+
+* Keyboard Maintainer: [Maartenwut](https://github.com/maartenwut)  
+* Hardware Supported: J-01 PCB  
+* Hardware Availability: Private Group Buy and [Charity Raffle](https://novelkeys.xyz/collections/frontpage/products/j-01-raffle-ticket)
```
Comment (fauxpark) [inline keyboards/jae/j01/readme.md]:
```suggestion
* Hardware Availability: Private Group Buy and [Charity Raffle](https://novelkeys.xyz/collections/frontpage/products/j-01-raffle-ticket)
```

File: keyboards/jae/j01/info.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "",
```
Comment (fauxpark) [inline keyboards/jae/j01/info.json]:
```suggestion
    "keyboard_name": "J01", 
```

File: keyboards/jae/j01/keymaps/default/config.h [hunk 6]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 Maarten Dekkers <maartenwut@gmail.com>
```
Comment (fauxpark) [inline keyboards/jae/j01/keymaps/default/config.h]: This file can be deleted

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**